iis/tests/test_e2e.py

```diff
@@ -0,0 +1,15 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+import pytest
+
+from datadog_checks.dev.testing import requires_py3
+from datadog_checks.iis import IIS
+
+
+@pytest.mark.e2e
+@requires_py3
+def test_e2e_py3(dd_agent_check, aggregator, instance, check):
```

Comment:
Is `check` being used here?

Reply:
We added it to get the aggregator to run the check, I'll test to see if it's not necessary

Reply:
Looks like it unnecessary, I removed it in the latest commit.

Reply:
@yzhan289 Let me know if we're good to go here!